examples/step-44/CMakeLists.txt

```diff
@@ -34,6 +34,17 @@ IF(NOT ${deal.II_FOUND})
     )
 ENDIF()
 
+#
+# Are all dependencies fulfilled?
+#
+IF(NOT DEAL_II_WITH_CXX11)
+  MESSAGE(FATAL_ERROR "
+Error! The deal.II library found at ${DEAL_II_PATH} was not configured with
+    DEAL_II_WITH_CXX11 = ON
+One or all of these are OFF in your installation but are required for this tutorial step."
```

Comment:
Hey @davydden . This message is taken verbatim from step-33's `CMakeLists.txt`. I kept it as is for consistency. I don't mind changing it, but then shouldn't it be changed for all of the tutorials that require a single dependency?